contracts/WETH10.sol

```diff
@@ -71,16 +134,20 @@ contract WETH10 {
         emit Transfer(msg.sender, address(0), value);
     }
     
+     /**
+     * @dev Burns an account's (`from`) WETH10 token `value` to withdraw matching ether to account (`to`).
+     * `value` is then deducted from the caller's allowance.
+     * 
+     * lock is provided for functions that deal with ether as check on potential abuse.
+     *
+     * Emits a {Transfer} event to reflect WETH10 token burn of `value` to zero address from account (`from`).
+     */
     function withdrawFrom(address from, address to, uint256 value) external lock {
         require(balanceOf[from] >= value, "!balance");
 
-        
```

Comment:
I'm not sure if you branched before pulling, but here you are undoing one of the recent PRs. The code in red is the right version.